### ISSUE TITLE
Make DID states downloadable

### ISSUE THREAD

**Issue by chris-tbd** [TBD54566975/web5-js#29]
Should be downloadable via `web5.did.download()` or something similar

**Comment by michaelneale** [comment]
yes, this would be good. DID + VCs. I wonder if this could also include a general DWN backup (but I may be scope creeping)

**Comment by michaelneale** [comment]
Note - it is trivial in page to ball up data in a zip for download. see this code example: https://gist.github.com/michaelneale/0804d19db3c98cb21e6f3af1c9741918 - this works on a phone as well where people can back up their state and then restore it (as per #74 )

**Comment by michaelneale** [comment]
I would think this should include DID (including all generated stuff) and perhaps... VCs?

**Comment by michaelneale** [comment]
this shows how it can be done: https://github.com/TBD54566975/web5-js/pull/127
